boards/sensry/ganymed_bob/doc/index.rst

```diff
@@ -0,0 +1,141 @@
+.. _ganymed_bob:
+
+Ganymed BOB
+############
+
+Overview
+********
+
+.. note::
+
+   All software for the nGanymed BOB is experimental and hardware availability
+   is restricted to the participants in the limited sampling program.
+
+The Ganymed BreakOut Board hardware provides
+support for the Ganymed SY1xx series IoT multicore RISC-V SoC with optional sensor level.
+
+The SoC has the following core features:
+
+* 32-Bit RSIC-V 9-core processor, up to 500MHz
+  * 1x Data Aquisition Unit
+  * 8x Data Processing Unit
+  * Event Bus
+  * MicroDMA
+* 4096 KB Global SRAM
+* 64 KB Secure SRAM
+* 512 KB Global MRAM
+* 512 KB Secure MRAM
+* CLOCK
+* RAM
+* :abbr:`32x GPIO (General Purpose Input Output)`
+* :abbr:`4x TWIM (I2C-compatible two-wire interface with MicroDMA)`
+* 4x I2S
+* :abbr:`7x SPI (Serial Peripheral Interface with MicroDMA)`
+* :abbr:`3x UART (Universal receiver-transmitter with MicroDMA)`
+* :abbr:`1x TSN (Time sensitive networking ethernet MAC with MicroDMA)`
+* 1x CAN-FD
+* 3x ADC
+
+.. figure:: img/ganymed_breakout_gbm.webp
+     :align: center
+     :alt: Ganymed BOB
+
+     Ganymed BOB with SY120-GBM(Credit: Sensry)
+
+.. figure:: img/ganymed_breakout_gen1.webp
+     :align: center
+     :alt: Ganymed BOB with SY120-GEN1
+
+     Ganymed BOB (Credit: Sensry)
+
+Hardware
+********
+
+The BreakOut Board has:
+
+* Assembled SoC of the Sensry Ganymed SY1xx series w/ (GENx) or w/o (GMB) top level sensors:
+ * SY120-GBM - Generic Base Module without top level sensors
+ * SY120-GEN1 - Generic Module with top level sensors (Bosch BME680 - SPI1 , Bosch BMA456 - SPI0, Bosch BMG250 - SPI2, STMicro MIS2DH - I2C0)
+* power section for on-board power generation (selectable)
+* 40-pin JTAG connector (compatible to Olimex ARM-JTAG-OCD-H)
+* USB over FTDI (connected to UART0)
+* Header for I/Os and additional configuration
+
+Supported Features
+==================
+
+The variant ``ganymed-bob/sy120-gbm/cpuapp`` board configuration supports the following
```

Comment:
*board target, the board target is also actually wrong

Reply:
fixed